rpc/wrtc_call_queue_mongodb_test.go

```diff
@@ -198,3 +207,29 @@ func TestMongoDBWebRTCCallQueueMulti(t *testing.T) {
 		test.That(t, foundHosts, test.ShouldBeTrue)
 	})
 }
+
+func addFakeAnswererForHost(t *testing.T, client *mongo.Client, host string) {
+	t.Helper()
+	_, err := client.Database(mongodbWebRTCCallQueueDBName).Collection(mongodbWebRTCCallQueueOperatorsCollName).InsertOne(context.Background(),
+		bson.M{
+			"_id":       operatorID,
+			"expire_at": time.Now().Add(time.Hour),
+			"hosts": []bson.M{{
+				"host":          host,
+				"caller_size":   int64(0),
+				"answerer_size": int64(1),
+			}},
+		})
+	test.That(t, err, test.ShouldBeNil)
+}
+
+func removeFakeAnswererForHost(t *testing.T, client *mongo.Client, host string) {
+	t.Helper()
+	_, err := client.Database(mongodbWebRTCCallQueueDBName).Collection(mongodbWebRTCCallQueueOperatorsCollName).UpdateOne(context.Background(),
```

Comment:
> UpdateOne

[q] What you have here seems fine, but is it necessary that this be an `UpdateOne` to decrement the answerer size rather than a `RemoveOne` that just removes the `operatorID` document?